src/main/java/io/jenkins/plugins/synopsys/security/scan/factory/ScanParametersFactory.java

```diff
@@ -472,4 +480,42 @@ public static boolean validateProduct(String product, TaskListener listener) {
 
         return isValid;
     }
+
+    public static Result getBuildResultIfIssuesAreFound(
+            int exitCode, String markBuildIfIssuesArePresent, LoggerWrapper logger) {
+        Result result = null;
+
+        if (!Utility.isStringNullOrBlank((markBuildIfIssuesArePresent))) {
+            if (exitCode == ErrorCode.BRIDGE_BUILD_BREAK) {
+                try {
+                    BuildStatus buildStatus = BuildStatus.valueOf(markBuildIfIssuesArePresent.toUpperCase());
+                    if (buildStatus.in(BuildStatus.FAILURE, BuildStatus.UNSTABLE, BuildStatus.SUCCESS)) {
+                        result = getMappedResultForBuildStatus(buildStatus);
+                    }
+                } catch (IllegalArgumentException e) {
+                    logger.warn("Unsupported value for build status: " + markBuildIfIssuesArePresent
+                            + ". Supported values are: "
+                            + List.of(BuildStatus.FAILURE, BuildStatus.UNSTABLE, BuildStatus.SUCCESS));
```

Comment:
- This warning message should be before `********** End Execution of Synopsys Security Scan ***********`
- Instead of `List.of(BuildStatus.FAILURE, BuildStatus.UNSTABLE, BuildStatus.SUCCESS)`, use this `Arrays.asList(BuildStatus.values())`